### ISSUE TITLE
database overview: list_models() and verbosity control?

### ISSUE THREAD

**Issue by gabrielastro** [tomasstolker/species#80]
Looking at https://species.readthedocs.io/en/latest/database.html, there is `database.list_content()` to print *everything*, and there exists `database.list_companions()`. But what about listing just the models available? Also it might be nice to have a verbosity flag to all these methods. The current output can be a bit massive (and thus not much of an overview), which is not always desired.

**Comment by tomasstolker** [comment]
Thanks for the suggestion! In commit 001aef6b9c1c990ea33057b3674ec58ddb30bfc7, I have added the `verbose` parameter to `list_companions` and `available_models`. The latter method of `Database` can be used to list the available model spectra. For `list_content` I find a verbose parameter not so useful.

**Comment by gabrielastro** [comment]
Ok, thanks! I tried it out and it is practical.

Tiny thing: strip the `/abstract` from the NASA ADS (just visual noise and not needed) :nerd_face:  and maybe uniformise to have an empty line after all entries, not only those with an URL:
```
      - Reference: Linder et al. (2019)
      - URL: https://ui.adsabs.harvard.edu/abs/2019A%26A...623A..85L/abstract

   - Saumon & Marley (2008) clear:
      - Label = saumon2008-clear
      - Model parameters: ['teff', 'logg']
      - Teff range (K): [500, 2400]
      - Wavelength range (um): [0.4, 50]
      - Resolution lambda/Dlambda: 5000
      - File size: 197 Mb
   - Saumon & Marley (2008) cloudy:
      - Label = saumon2008-cloudy
      - Model parameters: ['teff', 'logg']
```
Well, actually, except maybe for the blackbody, I guess every entry needs a URL for the reference :grin:.

**Comment by tomasstolker** [comment]
Thanks for testing! The other suggestions have been fixed in commit bc6a4d3a4129fb792a57c7ce0b804232e1d8c9ec

**Comment by gabrielastro** [comment]
Excellent, thank you!
